python/ray/rllib/models/catalog.py

```diff
@@ -75,7 +76,11 @@ def ConvolutionalNetwork(inputs, num_outputs, options=None):
         return ConvolutionalNetwork(inputs, num_outputs, options)
 
     @staticmethod
-    def get_preprocessor(env_name, obs_shape):
+    def LSTM(inputs, num_outputs, options=None):
```

Comment:
Hm I'm not sure about putting LSTM in here like this, could we keep it A3C-specific for now?